public/wp-content/themes/glific/page-faq.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * File functions.php for theme glific
+ *
+ * @package glific
+ */
+
+get_header();
+
+$faqs = get_field('faq_content');
+$page_title = get_field('faq_page_title');
+?>
+
+
+<div class="bg-theme-white-smoke page-faq">
+    <div class="demo-section d-flex flex-column flex-md-row justify-content-md-between w-320 w-md-660 w-xl-880 mx-auto">
+        <div class="d-flex flex-column w-full justify-content-center pt-18 text-center">
+            <h3 class="text-theme-primary font-heebo-bold fz-28 fz-md-36 leading-40 mb-0"><?php echo $page_title; ?></h3>
```

Comment:
Looks like, you are using the spacer in place tab for indentation. WordPress takes tab rather than the spacer. Please format it with a tab. Formate the  whole file